packages/ssr-plugin/example/public/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { h, render } from 'preact';
+import { LocationProvider, Router, ErrorBoundary, lazy, hydrate } from 'preact-iso';
+import Home from './pages/home.js';
+import NotFound from './pages/_404.js';
+import Header from './header.tsx';
+
+const About = lazy(() => import('./pages/about/index.js'));
+const LazyAndLate = lazy(
+	() =>
+		new Promise(r => {
+			setTimeout(() => {
+				r(import('./pages/about/index.js'));
+			}, 1.5e3);
+		})
+);
+const CompatPage = lazy(() => import('./pages/compat.js'));
+const ClassFields = lazy(() => import('./pages/class-fields.js'));
+const Files = lazy(() => import('./pages/files/index.js'));
+const Environment = lazy(async () => (await import('./pages/environment/index.js')).Environment);
+
+export function App() {
+	return (
+		<LocationProvider>
+			<div class="app">
+				<Header />
+				<ErrorBoundary>
+					<Router>
+						<Home path="/" />
+						<About path="/about" />
+						<LazyAndLate path="/lazy-and-late" title={'Lazy and Late'} />
+						<CompatPage path="/compat" />
+						<ClassFields path="/class-fields" />
+						<Files path="/files" />
+						<Environment path="/env" />
+						<NotFound default />
+					</Router>
+				</ErrorBoundary>
+			</div>
+		</LocationProvider>
+	);
+}
+
+if (typeof window !== 'undefined') {
+	hydrate(<App />, document.body);
+}
```

Comment:
You will probably see this in PR #403 ( https://github.com/preactjs/wmr/pull/403#pullrequestreview-605831104 ) but just in case this goes under the radar for this PR #400, here it is again:

The `typeof window !== 'undefined'` browser check can be removed.

References:

https://github.com/preactjs/wmr/blob/fe51b7eb3a65740571863fbeebf3367b74a9ddfe/packages/preact-iso/hydrate.js#L7

https://github.com/preactjs/wmr/blob/main/packages/preact-iso/README.md#hydratejs
> This method also checks to make sure its running in a browser context before attempting any rendering - if not, it does nothing.